[Automation] native-image run fails for org.flywaydb:flyway-database-postgresql:10.19.0

## native-image run on ubuntu-latest / JDK 25

Failure kind: native-image run
OS: ubuntu-latest
JDK: 25
Reproducer: `GVM_TCK_LV="10.19.0" ./gradlew clean test -Pcoordinates="org.flywaydb:flyway-database-postgresql:10.13.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/23928244621/job/69789708439
Last 300 lines of the log (excerpt truncated to fit the GitHub issue size limit):
```console
f=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.platform=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.ref=org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=java.base/jdk.internal.reflect=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.util=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.vm.annotation=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.vm=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.invoke.util=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.net.www=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.net=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.nio.ch=org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=java.base/sun.reflect.annotation=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.factory=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.reflectiveObjects=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.repository=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.scope=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.tree=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.jca=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.provider=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.rsa=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.ssl=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.util=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.x509=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.text.spi=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.calendar=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.cldr=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.locale.provider=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.locale=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.resources=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util=org.graalvm.nativeimage.builder \
--add-exports=java.management/com.sun.jmx.mbeanserver=org.graalvm.nativeimage.builder \
--add-exports=java.management/sun.management=org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.graal.compiler/jdk.graal.compiler.util.json=com.oracle.graal.reporter \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.aarch64=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.amd64=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.site=com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.stack=jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.truffle.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.common=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.aarch64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.amd64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.riscv64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot=com.oracle.graal.graal_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.meta=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.truffle.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.riscv64=com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.runtime=com.oracle.graal.graal_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.services=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.events=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.event=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.jfc=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.management/com.sun.management.internal=org.graalvm.nativeimage.builder \
-XX:+UseJVMCINativeLibrary \
-Xss10m \
-XX:+UseParallelGC \
-XX:MaxRAMPercentage=85.0 \
'-Dsvm.build.memoryUsageReasonText=$CI set to '"'"'true'"'"'' \
-XX:GCTimeRatio=9 \
-XX:+ExitOnOutOfMemoryError \
-Djava.awt.headless=true \
'-Dorg.graalvm.vendor=Oracle Corporation' \
-Dorg.graalvm.vendorurl=https://www.graalvm.org/ \
'-Dorg.graalvm.vendorversion=Oracle GraalVM 25.0.2+10.1' \
-Dorg.graalvm.version=25.0.2 \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Dorg.graalvm.nativeimage.imagecode=buildtime \
-Xshare:off \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-Djava.lang.invoke.MethodHandle.DONT_INLINE_THRESHOLD=-1 \
-Djava.lang.invoke.MethodHandle.PROFILE_GWT=false \
-Djunit.platform.listeners.uid.tracking.output.dir=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist \
--add-modules=ALL-DEFAULT \
--enable-native-access=org.graalvm.nativeimage.shaded.capnproto,org.graalvm.nativeimage.configure,org.graalvm.espresso.shared.svm,com.oracle.svm.svm_enterprise,com.oracle.graal.reporter,org.graalvm.nativeimage.foreign,org.graalvm.nativeimage.pointsto,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.base,org.graalvm.nativeimage.objectfile \
--module-path \
/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/espresso-svm.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/svm.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/pointsto.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/reporter.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/svm-capnproto-runtime.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/svm-configure.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/objectfile.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/svm-foreign.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/native-image-base.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/builder/svm-enterprise.jar \
-Dsvm.modulesupport.addedModules=java.rmi,java.sql,org.graalvm.collections,jdk.jdi,java.logging,java.xml.crypto,java.transaction.xa,java.xml,jdk.jfr,jdk.internal.vm.ci,java.datatransfer,org.graalvm.truffle.compiler,jdk.internal.md,java.desktop,java.naming,jdk.internal.ed,java.prefs,java.net.http,jdk.compiler,java.security.sasl,jdk.internal.opt,org.graalvm.nativeimage,jdk.zipfs,jdk.attach,java.base,jdk.management.agent,org.graalvm.word,jdk.internal.le,java.management,java.sql.rowset,jdk.unsupported,jdk.jlink,jdk.jdwp.agent,jdk.internal.jvmstat,java.instrument,java.security.jgss,jdk.management,java.compiler,java.scripting,jdk.graal.compiler,jdk.jdeps,java.management.rmi \
--add-modules=java.rmi,java.sql,org.graalvm.collections,jdk.jdi,java.logging,java.xml.crypto,java.transaction.xa,java.xml,jdk.jfr,jdk.internal.vm.ci,java.datatransfer,org.graalvm.truffle.compiler,jdk.internal.md,java.desktop,java.naming,jdk.internal.ed,java.prefs,java.net.http,jdk.compiler,java.security.sasl,jdk.internal.opt,org.graalvm.nativeimage,jdk.zipfs,jdk.attach,java.base,jdk.management.agent,org.graalvm.word,jdk.internal.le,java.management,java.sql.rowset,jdk.unsupported,jdk.jlink,jdk.jdwp.agent,jdk.internal.jvmstat,java.instrument,java.security.jgss,jdk.management,java.compiler,java.scripting,jdk.graal.compiler,jdk.jdeps,java.management.rmi \
--module \
org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner \
-keepalive \
/proc/8674/comm \
-imagecp \
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/libs/org.flywaydb.flyway-database-postgresql_tests.jar:/home/runner/.m2/repository/org/graalvm/buildtools/junit-platform-native/1.0.1-SNAPSHOT/junit-platform-native-1.0.1-SNAPSHOT.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter/5.9.2/26c586fbe0ebd81b48c9f11f0d998124248697ae/junit-jupiter-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-reporting/1.9.2/8bea292e2a546e4c21071f6bee60917ffd9f0101/junit-platform-reporting-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-launcher/1.9.2/38e0bfad8c57d4cd1a8f27926c25ffe9543068d6/junit-platform-launcher-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/5.9.2/53421816bde124a564a64ba005dcc0c8e66a9722/junit-vintage-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/5.9.2/bc2765afb7b85b583c710dd259a11c6b8c39e912/junit-jupiter-params-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/5.9.2/572f7a553b53f83ee59cc045ce1c3772864ab76c/junit-jupiter-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/5.9.2/fed843581520eac594bc36bb4b0f55e7b947dda9/junit-jupiter-api-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/1.9.2/40aeef2be7b04f96bb91e8b054affc28b7c7c935/junit-platform-engine-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/1.9.2/6f9f8621d8230cd38aa42e58ccbc0c00569131ce/junit-platform-commons-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-console/1.9.2/5548148dd55e820c42bd817ab93c1eaa05ce37f6/junit-platform-console-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.assertj/assertj-core/3.22.0/c300c0c6a24559f35fa0bd3a5472dc1edcd0111e/assertj-core-3.22.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.awaitility/awaitility/4.2.0/2c39784846001a9cffd6c6b89c78de62c0d80fb8/awaitility-4.2.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.flywaydb/flyway-database-postgresql/10.19.0/3a263ca0b01009e07fba8810f611ce31574cbb24/flyway-database-postgresql-10.19.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.postgresql/postgresql/42.3.4/4fc2f1f47f685cf06ba684564ec6be8ed18791a7/postgresql-42.3.4.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest/2.1/9420ba32c29217b54eebd26ff7f9234d31c3fbb2/hamcrest-2.1.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.flywaydb/flyway-core/10.19.0/6ee0019a541e086c3a691bef290c0b8a1b74f4f4/flyway-core-10.19.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/3.5.0/2f50520c8abea66fbd8d26e481d3aef5c673b510/checker-qual-3.5.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.15.2/30d16ec2aef6d8094c5e2dce1d95034ca8b6cb42/jackson-datatype-jsr310-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.15.2/9353b021f10c307c00328f52090de2bdb4b6ff9c/jackson-databind-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.15.2/a6fe1836469a69b3ff66037c324d75fc66ef137c/jackson-core-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.15.2/4724a65ac8e8d156a24898d50fd5dbd3642870b8/jackson-annotations-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.dataformat/jackson-dataformat-toml/2.15.2/eb495841634034db2c635519c4048e96188bdc36/jackson-dataformat-toml-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.2.0/28c11eb91f9b6d8e200631d46e20a7f407f2a046/opentest4j-1.2.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/classes/java/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/resources/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist \
-imagemp \
/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/library-support.jar \
-H:CLibraryPath=/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/clibraries/linux-amd64/glibc,/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/clibraries/linux-amd64,/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/clibraries \
-H:Path@driver=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile \
-H:BuildOutputJSONFile@user=/tmp/native-image-build-output.json \
-H:ResourceConfigurationResources@file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/resources/test/META-INF/native-image/test/resource-config.json+api=META-INF/native-image/test/resource-config.json \
-H:FallbackThreshold@user+api=0 \
-H:Name@user+api=org.flywaydb.flyway-database-postgresql_tests-tests \
-H:ConfigurationFileDirectories@user=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/generated/generateTestResourcesConfigFile,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.flywaydb/flyway-database-postgresql/10.13.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.flywaydb/flyway-core/10.15.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-databind/2.15.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.postgresql/postgresql/42.3.4 \
-H:Optimize@user+api=b \
-H:+ReportExceptionStackTraces@user \
-H:Features@user+api=org.graalvm.junit.platform.JUnitPlatformFeature \
'-H:Class@explicit main-class=org.graalvm.junit.platform.NativeImageJUnitLauncher' \
-H:ImageBuildID@driver=a8ee2c76-252c-b884-c657-2039c9d7ee17 \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/thirdparty/native-image.properties+api=com.oracle.svm.thirdparty.gson.GsonFeature' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.2+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/polyglot/native-image.properties+api=com.oracle.svm.polyglot.groovy.GroovyIndyInterfaceFeature,com.oracle.svm.polyglot.scala.ScalaFeature'
]
========================================================================================================================
GraalVM Native Image: Generating 'org.flywaydb.flyway-database-postgresql_tests-tests' (executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------
[1/8] Initializing...                                                                                    (6.9s @ 0.17GB)
 Java version: 25.0.2+10-LTS, vendor version: Oracle GraalVM 25.0.2+10.1
 Graal compiler: optimization level: b, target machine: x86-64-v3, PGO: off
 C compiler: gcc (linux, x86_64, 13.3.0)
 Garbage collector: Serial GC (max heap size: 80% of RAM)
 2 user-specific feature(s):
 - com.oracle.svm.thirdparty.gson.GsonFeature
 - org.graalvm.junit.platform.JUnitPlatformFeature
------------------------------------------------------------------------------------------------------------------------
 1 experimental option(s) unlocked:
 - '-H:BuildOutputJSONFile' (origin(s): command line)
------------------------------------------------------------------------------------------------------------------------
Build resources:
 - 12.67GB of memory (75.6% of system memory, $CI set to 'true')
 - 4 thread(s) (100.0% of 4 available processor(s), determined at start)
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist] and its subfolders.
[2/8] Performing analysis...  [*****]                                                                   (54.1s @ 1.39GB)
   12,439 types,  16,898 fields, and  60,636 methods found reachable
    4,029 types,     174 fields, and   2,459 methods registered for reflection
       62 types,      63 fields, and      55 methods registered for JNI access
        0 downcalls and 0 upcalls registered for foreign access
        4 native libraries: dl, pthread, rt, z
[3/8] Building universe...                                                                               (7.3s @ 1.34GB)
[4/8] Parsing methods...      [**]                                                                       (5.0s @ 1.46GB)
[5/8] Inlining methods...     [***]                                                                      (3.0s @ 1.56GB)
[6/8] Compiling methods...    [*****]                                                                   (31.0s @ 1.08GB)
[7/8] Laying out methods...   [**]                                                                       (4.8s @ 1.42GB)
[8/8] Creating image...       [***]                                                                      (5.1s @ 1.64GB)
  19.71MB (42.02%) for code area:    38,926 compilation units
  21.23MB (45.27%) for image heap:  315,334 objects and 165 resources
   5.96MB (12.71%) for other data
  46.90MB in total image size, 43.52MB in total file size
------------------------------------------------------------------------------------------------------------------------
Top 10 origins of code area:                                Top 10 object types in image heap:
   8.06MB java.base                                            5.85MB byte[] for code metadata
   3.37MB svm.jar (Native Image)                               3.83MB byte[] for java.lang.String
   2.67MB java.xml                                             2.04MB java.lang.String
 914.68kB jackson-databind-2.15.2.jar                          1.63MB java.lang.Class
 856.08kB postgresql-42.3.4.jar                                1.29MB com.oracle.svm.core.hub.DynamicHubCompanion
 576.14kB flyway-core-10.19.0.jar                              1.09MB byte[] for general heap data
 556.61kB assertj-core-3.22.0.jar                            472.86kB java.util.concurrent.ConcurrentHashMap$Node
 260.25kB jackson-core-2.15.2.jar                            400.00kB java.util.HashMap$Node
 236.45kB java.rmi                                           363.33kB c.o.svm.core.hub.DynamicHub$ReflectionMetadata
 197.79kB junit-jupiter-engine-5.9.2.jar                     322.83kB java.lang.Object[]
   1.40MB for 38 more packages                                 3.94MB for 3081 more object types
                            Use '--emit build-report' to create a report with more details.
------------------------------------------------------------------------------------------------------------------------
Security report:
 - Binary includes Java deserialization.
 - CycloneDX SBOM with 24 component(s) is embedded in binary (1.24kB). 2 type(s) could not be associated to a component.
 - Advanced obfuscation not enabled; enable with '-H:AdvancedObfuscation=""' (experimental support).
------------------------------------------------------------------------------------------------------------------------
Recommendations:
 G1GC: Use the G1 GC ('--gc=G1') for improved latency and throughput.
 PGO:  Use Profile-Guided Optimizations ('--pgo') for improved throughput.
 FUTR: Use '--future-defaults=all' to prepare for future releases.
 HEAP: Set max heap for improved and more predictable memory usage.
 CPU:  Enable more CPU features with '-march=native' for improved performance.
------------------------------------------------------------------------------------------------------------------------
                       9.1s (7.5% of total time) in 1295 GCs | Peak RSS: 2.48GB | CPU load: 3.63
------------------------------------------------------------------------------------------------------------------------
Build artifacts:
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile/org.flywaydb.flyway-database-postgresql_tests-tests (executable)
 /tmp/native-image-build-output.json (build_info)
========================================================================================================================
Finished generating 'org.flywaydb.flyway-database-postgresql_tests-tests' in 1m 59s.
[native-image-plugin] Native Image written to: /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile

> Task :nativeTest FAILED
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist] and its subfolders.
JUnit Platform on Native Image - report
----------------------------------------

Starting PostgreSQL ...
PostgreSQL started
Shutting down PostgreSQL
flyway.database.postgresql.FlywayDatabasePostgresqlTests > migrate() FAILED


Failures (1):
  JUnit Jupiter:FlywayDatabasePostgresqlTests:migrate()
    MethodSource [className = 'flyway.database.postgresql.FlywayDatabasePostgresqlTests', methodName = 'migrate', methodParameterTypes = '']
    => org.flywaydb.core.api.FlywayException: No serializer found for class org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS)
       org.flywaydb.core.extensibility.ConfigurationExtension.copy(ConfigurationExtension.java:45)
       java.base@25.0.2/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:214)
       java.base@25.0.2/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1716)
       java.base@25.0.2/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
       java.base@25.0.2/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
       [...]
     Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS)
       com.fasterxml.jackson.databind.SerializerProvider.reportBadDefinition(SerializerProvider.java:1308)
       com.fasterxml.jackson.databind.DatabindContext.reportBadDefinition(DatabindContext.java:414)
       com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.failForEmpty(UnknownSerializer.java:53)
       com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.serialize(UnknownSerializer.java:30)
       com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:479)
       [...]

Test run finished after 1543 ms
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         0 tests successful      ]
[         1 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeTest'.
> Process 'command '/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile/org.flywaydb.flyway-database-postgresql_tests-tests'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 2m 8s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 7 executed, 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.flywaydb:flyway-database-postgresql:10.13.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2m 10s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 10.19.0 failed with exit code 1.
FAILED[native-image run][10.19.0][GVM_TCK_LV="10.19.0" ./gradlew clean test -Pcoordinates="org.flywaydb:flyway-database-postgresql:10.13.0"]
```

## native-image run on ubuntu-latest / JDK latest-ea

Failure kind: native-image run
OS: ubuntu-latest
JDK: latest-ea
Reproducer: `GVM_TCK_LV="10.19.0" ./gradlew clean test -Pcoordinates="org.flywaydb:flyway-database-postgresql:10.13.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/23928244621/job/69789708443
Last 300 lines of the log (excerpt truncated to fit the GitHub issue size limit):
```console
ge.objectfile \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.amd64=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.site=com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.stack=jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,jdk.graal.compiler.hostvmaccess,jdk.graal.compiler.vmaccess,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.truffle.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.common=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.aarch64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.amd64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.riscv64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot=com.oracle.graal.graal_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.meta.annotation=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,jdk.graal.compiler.hostvmaccess,jdk.graal.compiler.vmaccess,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.guest.staging,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.meta=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,jdk.graal.compiler.hostvmaccess,jdk.graal.compiler.vmaccess,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.guest.staging,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.truffle.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.riscv64=com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.runtime=com.oracle.graal.graal_enterprise,jdk.graal.compiler,jdk.graal.compiler.hostvmaccess,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.services=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.events=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.event=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.jfc=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.settings=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.tracing=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.management/com.sun.management.internal=org.graalvm.nativeimage.builder \
-XX:+UseJVMCINativeLibrary \
-XX:MaxRAMPercentage=84.99999999880707 \
'-Dsvm.build.memoryUsageReasonText=13.27GiB of memory (85% of system memory because $CI set to '"'"'true'"'"')' \
-Xss10m \
-XX:+UseParallelGC \
-XX:GCTimeRatio=9 \
-XX:+ExitOnOutOfMemoryError \
-XX:+UseCompressedOops \
-Djava.awt.headless=true \
'-Dorg.graalvm.vendor=Oracle Corporation' \
-Dorg.graalvm.vendorurl=https://www.graalvm.org/ \
'-Dorg.graalvm.vendorversion=Oracle GraalVM 25.1.0-dev+10.1' \
-Dorg.graalvm.version=25.1.0-dev \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Djava.util.concurrent.ForkJoinPool.common.exceptionHandler=com.oracle.svm.hosted.CommonPoolUncaughtExceptionHandler \
'-Djava.util.concurrent.ForkJoinPool.common.threadFactory=com.oracle.svm.hosted.NativeImageSystemClassLoader$NativeImageForkJoinWorkerThreadFactory' \
-Dorg.graalvm.nativeimage.imagecode=buildtime \
-Xshare:off \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-Djava.lang.invoke.MethodHandle.DONT_INLINE_THRESHOLD=-1 \
-Djava.lang.invoke.MethodHandle.PROFILE_GWT=false \
-Djunit.platform.listeners.uid.tracking.output.dir=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist \
-Dcom.oracle.svm.provided=/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest-staging.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-shared.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/vmaccess-guest.jar \
--add-modules=ALL-DEFAULT \
--enable-native-access=org.graalvm.nativeimage.objectfile,com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.shaded.capnproto,com.oracle.graal.reporter,jdk.graal.compiler.vmaccess.guest,org.graalvm.nativeimage.guest.staging,jdk.graal.compiler.hostvmaccess,jdk.graal.compiler.vmaccess,org.graalvm.nativeimage.foreign,org.graalvm.nativeimage.builder,org.graalvm.espresso.shared.svm,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.base,org.graalvm.nativeimage.shared,org.graalvm.nativeimage.pointsto \
--module-path \
/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/espresso-svm.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/pointsto.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-guest-staging.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/reporter.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-capnproto-runtime.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-configure.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-shared.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/objectfile.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/hostvmaccess.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-foreign.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/vmaccess.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/native-image-base.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-enterprise.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/vmaccess-guest.jar \
--module \
org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner \
-keepalive \
/proc/8462/comm \
-imagecp \
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/libs/org.flywaydb.flyway-database-postgresql_tests.jar:/home/runner/.m2/repository/org/graalvm/buildtools/junit-platform-native/1.0.1-SNAPSHOT/junit-platform-native-1.0.1-SNAPSHOT.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter/5.9.2/26c586fbe0ebd81b48c9f11f0d998124248697ae/junit-jupiter-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-reporting/1.9.2/8bea292e2a546e4c21071f6bee60917ffd9f0101/junit-platform-reporting-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-launcher/1.9.2/38e0bfad8c57d4cd1a8f27926c25ffe9543068d6/junit-platform-launcher-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/5.9.2/53421816bde124a564a64ba005dcc0c8e66a9722/junit-vintage-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/5.9.2/bc2765afb7b85b583c710dd259a11c6b8c39e912/junit-jupiter-params-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/5.9.2/572f7a553b53f83ee59cc045ce1c3772864ab76c/junit-jupiter-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/5.9.2/fed843581520eac594bc36bb4b0f55e7b947dda9/junit-jupiter-api-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/1.9.2/40aeef2be7b04f96bb91e8b054affc28b7c7c935/junit-platform-engine-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/1.9.2/6f9f8621d8230cd38aa42e58ccbc0c00569131ce/junit-platform-commons-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-console/1.9.2/5548148dd55e820c42bd817ab93c1eaa05ce37f6/junit-platform-console-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.assertj/assertj-core/3.22.0/c300c0c6a24559f35fa0bd3a5472dc1edcd0111e/assertj-core-3.22.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.awaitility/awaitility/4.2.0/2c39784846001a9cffd6c6b89c78de62c0d80fb8/awaitility-4.2.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.flywaydb/flyway-database-postgresql/10.19.0/3a263ca0b01009e07fba8810f611ce31574cbb24/flyway-database-postgresql-10.19.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.postgresql/postgresql/42.3.4/4fc2f1f47f685cf06ba684564ec6be8ed18791a7/postgresql-42.3.4.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest/2.1/9420ba32c29217b54eebd26ff7f9234d31c3fbb2/hamcrest-2.1.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.flywaydb/flyway-core/10.19.0/6ee0019a541e086c3a691bef290c0b8a1b74f4f4/flyway-core-10.19.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/3.5.0/2f50520c8abea66fbd8d26e481d3aef5c673b510/checker-qual-3.5.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.15.2/30d16ec2aef6d8094c5e2dce1d95034ca8b6cb42/jackson-datatype-jsr310-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.15.2/9353b021f10c307c00328f52090de2bdb4b6ff9c/jackson-databind-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.15.2/a6fe1836469a69b3ff66037c324d75fc66ef137c/jackson-core-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.15.2/4724a65ac8e8d156a24898d50fd5dbd3642870b8/jackson-annotations-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.dataformat/jackson-dataformat-toml/2.15.2/eb495841634034db2c635519c4048e96188bdc36/jackson-dataformat-toml-2.15.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.2.0/28c11eb91f9b6d8e200631d46e20a7f407f2a046/opentest4j-1.2.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/classes/java/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/resources/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist \
-imagemp \
/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar \
-H:CLibraryPath=/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries/linux-amd64/glibc,/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries/linux-amd64,/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries \
-H:Path@driver=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile \
-H:BuildOutputJSONFile@user=/tmp/native-image-build-output.json \
-H:ResourceConfigurationResources@file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/resources/test/META-INF/native-image/test/resource-config.json+api=META-INF/native-image/test/resource-config.json \
-H:FallbackThreshold@user+api=0 \
-H:Name@user+api=org.flywaydb.flyway-database-postgresql_tests-tests \
-H:ConfigurationFileDirectories@user=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/generated/generateTestResourcesConfigFile,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.flywaydb/flyway-database-postgresql/10.13.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.flywaydb/flyway-core/10.15.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-databind/2.15.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.postgresql/postgresql/42.3.4 \
-H:Optimize@user+api=b \
-H:+ReportExceptionStackTraces@user \
-H:Features@user+api=org.graalvm.junit.platform.JUnitPlatformFeature \
'-H:Class@explicit main-class=org.graalvm.junit.platform.NativeImageJUnitLauncher' \
-H:ImageBuildID@driver=a8ee2c76-252c-b884-c657-2039c9d7ee17 \
'-H:AddExports@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=java.base/jdk.internal.misc=org.graalvm.nativeimage.guest' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=com.oracle.svm.guest.features.jdk.hosted.JavaNetHttpFeature' \
'-H:ClassInitialization@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=sun.net.httpserver.simpleserver:run_time' \
'-H:ReachabilityMetadataResources@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/reachability-metadata.json+api=META-INF/native-image/org.graalvm.nativeimage.guest/reachability-metadata.json' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/thirdparty/native-image.properties+api=com.oracle.svm.thirdparty.gson.GsonFeature' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/polyglot/native-image.properties+api=com.oracle.svm.polyglot.groovy.GroovyIndyInterfaceFeature' \
-H:DriverWarningsCount=3
]
Warning: Option 'FallbackThreshold' is deprecated and might be removed in a future release: It no longer has any effect, and no replacement is available. Please refer to the GraalVM release notes.
========================================================================================================================
GraalVM Native Image: Generating 'org.flywaydb.flyway-database-postgresql_tests-tests' (executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------
[1/8] Initializing...                                                                                   (6.8s @ 0.22GiB)
 Builder configuration:
 - Java version: 25.0.2+10-LTS, vendor version: Oracle GraalVM 25.1.0-dev+10.1
 - Graal compiler: optimization level: b, target machine: x86-64-v3, PGO: off
 - C compiler: gcc (linux, x86_64, 13.3.0)
 - Assertions: disabled, system assertions: disabled
 - 1 user-specific feature(s):
   - org.graalvm.junit.platform.JUnitPlatformFeature
 Image configuration:
 - Garbage collector: Serial GC (max heap size: 80% of RAM)
 - Assertions: disabled (class-specific config may apply), system assertions: disabled
------------------------------------------------------------------------------------------------------------------------
 1 experimental option(s) unlocked:
 - '-H:BuildOutputJSONFile' (origin(s): command line)
------------------------------------------------------------------------------------------------------------------------
Build resources:
 - 13.27GiB of memory (85% of system memory because $CI set to 'true')
 - 4 thread(s) (100.0% of 4 available processor(s), determined at start)
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist] and its subfolders.
[2/8] Performing analysis...  [******]                                                                 (49.6s @ 1.36GiB)
   12,725 types,  16,995 fields, and  61,630 methods found reachable
      579 types,     101 fields, and   1,680 methods registered for reflection
       62 types,      63 fields, and      55 methods registered for JNI access
      166 resource accesses registered with 264.03KiB total size
        4 native libraries: dl, pthread, rt, z
[3/8] Building universe...                                                                              (7.1s @ 1.30GiB)
[4/8] Parsing methods...      [**]                                                                      (4.9s @ 1.41GiB)
[5/8] Inlining methods...     [***]                                                                     (2.7s @ 1.50GiB)
[6/8] Compiling methods...    [*****]                                                                  (29.3s @ 1.05GiB)
[7/8] Laying out methods...   [**]                                                                      (4.7s @ 1.39GiB)
[8/8] Creating image...       [**]                                                                      (4.8s @ 1.58GiB)
  17.18MiB (38.00%) for code area:    39,539 compilation units
  22.19MiB (49.07%) for image heap:  366,136 objects and 30 resources
   5.85MiB (12.93%) for other data
  45.22MiB in total image size, 41.94MiB in total file size
------------------------------------------------------------------------------------------------------------------------
Top 10 origins of code area:                                Top 10 object types in image heap:
   1.73MiB o.g.n.runtime/com.oracle.svm.core.code              5.43MiB byte[] for code metadata
 848.77KiB java.base/sun.security.ssl                          3.90MiB byte[] for string data
 766.45KiB java.base/java.util                                 2.12MiB java.base/java.lang.String
 458.25KiB java.base/com.sun.crypto.provider                   1.58MiB java.base/java.lang.Class
 407.58KiB java.base/jdk.internal.classfile.impl               1.36MiB o.g.n.runtime/c.o.s.core.hub.DynamicHubCompanion
 386.35KiB java.base/java.lang                               995.45KiB byte[] for general heap data
 344.55KiB java.base/java.lang.invoke                        597.34KiB java.base/j.u.concurrent.ConcurrentHashMap$Node
 306.81KiB java.xml/com.sun.org.apache.xerces.internal.impl  555.44KiB java.base/java.util.HashMap$Node
 292.47KiB java.xml/c.s.o.a.x.internal.impl.xs.traversers    534.83KiB byte[] for reflection metadata
 290.97KiB java.base/java.util.concurrent                    493.09KiB o.g.n.runtime/c.o.s.c.h.ImageReflectionMetadata
  10.86MiB for 492 more packages                               4.69MiB for 3181 more object types
                            Use '--emit build-report' to create a report with more details.
------------------------------------------------------------------------------------------------------------------------
Security report:
 - Binary includes Java deserialization.
 - CycloneDX SBOM with 25 component(s) is embedded in binary (1.35KiB). 2 type(s) could not be associated to a component.
 - Advanced obfuscation not enabled; enable with '-H:AdvancedObfuscation=""' (experimental support).
------------------------------------------------------------------------------------------------------------------------
Recommendations:
 G1GC: Use the G1 GC ('--gc=G1') for improved latency and throughput.
 PGO:  Use Profile-Guided Optimizations ('--pgo') for improved throughput.
 FUTR: Use '--future-defaults=all' to prepare for future releases.
 HEAP: Set max heap for improved and more predictable memory usage.
 CPU:  Enable more CPU features with '-march=native' for improved performance.
------------------------------------------------------------------------------------------------------------------------
                       9.6s (8.4% of total time) in 1255 GCs | Peak RSS: 2.28GiB | CPU load: 3.64
------------------------------------------------------------------------------------------------------------------------
Build artifacts:
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile/org.flywaydb.flyway-database-postgresql_tests-tests (executable, 41.94MiB)
 /tmp/native-image-build-output.json (build_info, 1.05KiB)
========================================================================================================================
Finished generating 'org.flywaydb.flyway-database-postgresql_tests-tests' in 1m 52s.

The build process encountered 4 warnings.

[native-image-plugin] Native Image written to: /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile

> Task :nativeTest FAILED
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/test-results/test/testlist] and its subfolders.
JUnit Platform on Native Image - report
----------------------------------------

Starting PostgreSQL ...
PostgreSQL started
Shutting down PostgreSQL
flyway.database.postgresql.FlywayDatabasePostgresqlTests > migrate() FAILED


Failures (1):
  JUnit Jupiter:FlywayDatabasePostgresqlTests:migrate()
    MethodSource [className = 'flyway.database.postgresql.FlywayDatabasePostgresqlTests', methodName = 'migrate', methodParameterTypes = '']
    => org.graalvm.nativeimage.MissingReflectionRegistrationError: Cannot reflectively invoke method 'public java.lang.String org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension.getDeployScript()'. To allow this operation, add the following to the 'reflection' section of 'reachability-metadata.json' and rebuild the native image:

    {
      "type": "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension",
      "methods": [
        {
          "name": "getDeployScript",
          "parameterTypes": []
        }
      ]
    }
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeTest'.
> Process 'command '/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/native/nativeTestCompile/org.flywaydb.flyway-database-postgresql_tests-tests'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 1m 59s


The 'reachability-metadata.json' file should be located in 'META-INF/native-image/<group-id>/<artifact-id>/' of your project. For further help, see https://www.graalvm.org/latest/reference-manual/native-image/metadata/#reflection
       org.graalvm.nativeimage.builder/com.oracle.svm.core.reflect.MissingReflectionRegistrationUtils.reportInvokedExecutable(MissingReflectionRegistrationUtils.java:110)
       java.base@25.0.2/java.lang.reflect.Method.acquireMethodAccessor(Method.java:151)
       java.base@25.0.2/java.lang.reflect.Method.invoke(Method.java:562)
       com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:688)
       com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:772)
       com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
       com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:479)
       com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:318)
       com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4719)
       com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3964)
       [...]

Test run finished after 1233 ms
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         0 tests successful      ]
[         1 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.flywaydb/flyway-database-postgresql/10.10.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 7 executed, 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.flywaydb:flyway-database-postgresql:10.13.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2m 1s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 10.19.0 failed with exit code 1.
FAILED[native-image run][10.19.0][GVM_TCK_LV="10.19.0" ./gradlew clean test -Pcoordinates="org.flywaydb:flyway-database-postgresql:10.13.0"]
```

--- a/metadata/org.flywaydb/flyway-database-postgresql/10.19.0/reachability-metadata.json
+++ b/metadata/org.flywaydb/flyway-database-postgresql/10.19.0/reachability-metadata.json
@@ -1,509 +1,497 @@
 {
-  "reflection": [
+  "reflection" : [
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "com.fasterxml.jackson.databind.ext.Java7SupportImpl",
-      "methods": [
+      "type" : "com.fasterxml.jackson.databind.ext.Java7SupportImpl",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.jdbc.JdbcUtils"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.jdbc.JdbcUtils"
       },
-      "type": "com.sun.crypto.provider.HmacCore$HmacSHA256",
-      "methods": [
+      "type" : "com.sun.crypto.provider.HmacCore$HmacSHA256",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.jdbc.JdbcUtils"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.jdbc.JdbcUtils"
       },
-      "type": "com.sun.crypto.provider.PBKDF2Core$HmacSHA256",
-      "methods": [
+      "type" : "com.sun.crypto.provider.PBKDF2Core$HmacSHA256",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "java.lang.Comparable"
+      "type" : "java.lang.Comparable"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "java.lang.String[]"
+      "type" : "java.lang.String[]"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.util.FeatureDetector"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.util.FeatureDetector"
       },
-      "type": "org.apache.commons.logging.Log"
+      "type" : "org.apache.commons.logging.Log"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.util.FeatureDetector"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.util.FeatureDetector"
       },
-      "type": "org.apache.logging.log4j.Logger"
+      "type" : "org.apache.logging.log4j.Logger"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineAppliedMigration"
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineAppliedMigration"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.api.configuration.ClassicConfiguration"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.api.configuration.ClassicConfiguration"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineMigrationConfigurationExtension"
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineMigrationConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineMigrationConfigurationExtension",
-      "methods": [
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineMigrationConfigurationExtension",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "getBaselineMigrationPrefix",
-          "parameterTypes": []
+          "name" : "getBaselineMigrationPrefix",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "setBaselineMigrationPrefix",
-          "parameterTypes": [
+          "name" : "setBaselineMigrationPrefix",
+          "parameterTypes" : [
             "java.lang.String"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineMigrationConfigurationExtension"
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineMigrationConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineMigrationResolver"
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineMigrationResolver"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineMigrationTypeResolver"
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineMigrationTypeResolver"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.api.migration.baseline.BaselineResourceTypeProvider"
+      "type" : "org.flywaydb.core.api.migration.baseline.BaselineResourceTypeProvider"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.experimental.migration.CoreMigrationTypeResolver"
+      "type" : "org.flywaydb.core.experimental.migration.CoreMigrationTypeResolver"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "type" : "org.flywaydb.core.extensibility.ConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.extensibility.Plugin"
+      "type" : "org.flywaydb.core.extensibility.Plugin"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.internal.command.clean.CleanModeConfigurationExtension",
-      "methods": [
+      "type" : "org.flywaydb.core.internal.command.clean.CleanModeConfigurationExtension",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "getClean",
-          "parameterTypes": []
+          "name" : "getClean",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "setClean",
-          "parameterTypes": [
+          "name" : "setClean",
+          "parameterTypes" : [
             "org.flywaydb.core.internal.command.clean.CleanModel"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.command.clean.CleanModeConfigurationExtension"
+      "type" : "org.flywaydb.core.internal.command.clean.CleanModeConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.internal.command.clean.CleanModel"
+      "type" : "org.flywaydb.core.internal.command.clean.CleanModel"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.internal.command.clean.SchemaModel"
+      "type" : "org.flywaydb.core.internal.command.clean.SchemaModel"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.api.configuration.ClassicConfiguration"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.api.configuration.ClassicConfiguration"
       },
-      "type": "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension"
+      "type" : "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension",
-      "methods": [
+      "type" : "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "getDeployScript",
-          "parameterTypes": []
+          "name" : "getDeployScript",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "setDeployScript",
-          "parameterTypes": [
+          "name" : "setDeployScript",
+          "parameterTypes" : [
             "java.lang.String"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension"
+      "type" : "org.flywaydb.core.internal.configuration.extensions.DeployScriptConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.configuration.resolvers.EnvironmentProvisionerNone"
+      "type" : "org.flywaydb.core.internal.configuration.resolvers.EnvironmentProvisionerNone"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.configuration.resolvers.EnvironmentVariableResolver"
+      "type" : "org.flywaydb.core.internal.configuration.resolvers.EnvironmentVariableResolver"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.database.base.TestContainersDatabaseType"
+      "type" : "org.flywaydb.core.internal.database.base.TestContainersDatabaseType"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.database.h2.H2DatabaseType"
+      "type" : "org.flywaydb.core.internal.database.h2.H2DatabaseType"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.database.sqlite.SQLiteDatabaseType"
+      "type" : "org.flywaydb.core.internal.database.sqlite.SQLiteDatabaseType"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.proprietaryStubs.AuthCommandExtensionStub"
+      "type" : "org.flywaydb.core.internal.proprietaryStubs.AuthCommandExtensionStub"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.proprietaryStubs.CommandExtensionStub"
+      "type" : "org.flywaydb.core.internal.proprietaryStubs.CommandExtensionStub"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.api.configuration.ClassicConfiguration"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.api.configuration.ClassicConfiguration"
       },
-      "type": "org.flywaydb.core.internal.proprietaryStubs.LicensingConfigurationExtensionStub"
+      "type" : "org.flywaydb.core.internal.proprietaryStubs.LicensingConfigurationExtensionStub"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.proprietaryStubs.LicensingConfigurationExtensionStub"
+      "type" : "org.flywaydb.core.internal.proprietaryStubs.LicensingConfigurationExtensionStub"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.api.configuration.ClassicConfiguration"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.api.configuration.ClassicConfiguration"
       },
-      "type": "org.flywaydb.core.internal.proprietaryStubs.PATTokenConfigurationExtensionStub"
+      "type" : "org.flywaydb.core.internal.proprietaryStubs.PATTokenConfigurationExtensionStub"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.proprietaryStubs.PATTokenConfigurationExtensionStub"
+      "type" : "org.flywaydb.core.internal.proprietaryStubs.PATTokenConfigurationExtensionStub"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.api.configuration.ClassicConfiguration"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.api.configuration.ClassicConfiguration"
       },
-      "type": "org.flywaydb.core.internal.publishing.PublishingConfigurationExtension"
+      "type" : "org.flywaydb.core.internal.publishing.PublishingConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.core.internal.publishing.PublishingConfigurationExtension",
-      "methods": [
+      "type" : "org.flywaydb.core.internal.publishing.PublishingConfigurationExtension",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "isPublishResult",
-          "parameterTypes": []
+          "name" : "isPublishResult",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "setPublishResult",
-          "parameterTypes": [
+          "name" : "setPublishResult",
+          "parameterTypes" : [
             "boolean"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.publishing.PublishingConfigurationExtension"
+      "type" : "org.flywaydb.core.internal.publishing.PublishingConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.resource.CoreResourceTypeProvider"
+      "type" : "org.flywaydb.core.internal.resource.CoreResourceTypeProvider"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.core.internal.schemahistory.BaseAppliedMigration"
+      "type" : "org.flywaydb.core.internal.schemahistory.BaseAppliedMigration"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.database.cockroachdb.CockroachDBDatabaseType"
+      "type" : "org.flywaydb.database.cockroachdb.CockroachDBDatabaseType"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.database.postgresql.PostgreSQLConfigurationExtension",
-      "methods": [
+      "type" : "org.flywaydb.database.postgresql.PostgreSQLConfigurationExtension",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "getTransactional",
-          "parameterTypes": []
+          "name" : "getTransactional",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "isTransactionalLock",
-          "parameterTypes": []
+          "name" : "isTransactionalLock",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "setTransactional",
-          "parameterTypes": [
+          "name" : "setTransactional",
+          "parameterTypes" : [
             "org.flywaydb.database.postgresql.TransactionalModel"
           ]
         },
         {
-          "name": "setTransactionalLock",
-          "parameterTypes": [
+          "name" : "setTransactionalLock",
+          "parameterTypes" : [
             "boolean"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.database.postgresql.PostgreSQLConfigurationExtension"
+      "type" : "org.flywaydb.database.postgresql.PostgreSQLConfigurationExtension"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "type": "org.flywaydb.database.postgresql.PostgreSQLDatabaseType"
+      "type" : "org.flywaydb.database.postgresql.PostgreSQLDatabaseType"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.extensibility.ConfigurationExtension"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.extensibility.ConfigurationExtension"
       },
-      "type": "org.flywaydb.database.postgresql.TransactionalModel",
-      "methods": [
+      "type" : "org.flywaydb.database.postgresql.TransactionalModel",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "getLock",
-          "parameterTypes": []
+          "name" : "getLock",
+          "parameterTypes" : [ ]
         },
         {
-          "name": "setLock",
-          "parameterTypes": [
+          "name" : "setLock",
+          "parameterTypes" : [
             "java.lang.Boolean"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.jdbc.JdbcUtils"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.jdbc.JdbcUtils"
       },
-      "type": "org.postgresql.Driver"
+      "type" : "org.postgresql.Driver"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.database.base.BaseDatabaseType"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.database.base.BaseDatabaseType"
       },
-      "type": "org.postgresql.jdbc.PgStatement"
+      "type" : "org.postgresql.jdbc.PgStatement"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.util.FeatureDetector"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.util.FeatureDetector"
       },
-      "type": "org.slf4j.Logger"
+      "type" : "org.slf4j.Logger"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.util.ClassUtils"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.util.ClassUtils"
       },
-      "type": "org.slf4j.spi.SLF4JServiceProvider"
+      "type" : "org.slf4j.spi.SLF4JServiceProvider"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.jdbc.JdbcUtils"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.jdbc.JdbcUtils"
       },
-      "type": "sun.security.provider.NativePRNG",
-      "methods": [
+      "type" : "sun.security.provider.NativePRNG",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": [
+          "name" : "<init>",
+          "parameterTypes" : [
             "java.security.SecureRandomParameters"
           ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.jdbc.JdbcUtils"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.jdbc.JdbcUtils"
       },
-      "type": "sun.security.provider.SHA2$SHA256",
-      "methods": [
+      "type" : "sun.security.provider.SHA2$SHA256",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     }
   ],
-  "resources": [
+  "resources" : [
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.logging.javautil.JavaUtilLog"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.logging.javautil.JavaUtilLog"
       },
-      "glob": "META-INF/services/java.time.zone.ZoneRulesProvider"
+      "glob" : "META-INF/services/java.time.zone.ZoneRulesProvider"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.plugin.PluginRegister"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.plugin.PluginRegister"
       },
-      "glob": "META-INF/services/org.flywaydb.core.extensibility.Plugin"
+      "glob" : "META-INF/services/org.flywaydb.core.extensibility.Plugin"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.scanner.ClasspathClassScanner"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.scanner.ClasspathClassScanner"
       },
-      "glob": "db/callback"
+      "glob" : "db/callback"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.scanner.Scanner"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.scanner.Scanner"
       },
-      "glob": "db/migration"
+      "glob" : "db/migration"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.resource.classpath.ClassPathResource"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.license.VersionPrinter"
       },
-      "glob": "db/migration/V1__create_table.sql"
+      "glob" : "org/flywaydb/core/internal/version.txt"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.resource.classpath.ClassPathResource"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.logging.javautil.JavaUtilLog"
       },
-      "glob": "db/migration/V2__alter_table.sql"
+      "module" : "java.logging",
+      "glob" : "sun/util/logging/resources/logging_en.properties"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.license.VersionPrinter"
+      "condition" : {
+        "typeReached" : "org.flywaydb.core.internal.logging.javautil.JavaUtilLog"
       },
-      "glob": "org/flywaydb/core/internal/version.txt"
+      "module" : "java.logging",
+      "glob" : "sun/util/logging/resources/logging_en_US.properties"
     },
     {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.logging.javautil.JavaUtilLog"
-      },
-      "module": "java.logging",
-      "glob": "sun/util/logging/resources/logging_en.properties"
-    },
-    {
-      "condition": {
-        "typeReached": "org.flywaydb.core.internal.logging.javautil.JavaUtilLog"
-      },
-      "module": "java.logging",
-      "glob": "sun/util/logging/resources/logging_en_US.properties"
-    },
-    {
-      "bundle": "sun.util.logging.resources.logging"
+      "bundle" : "sun.util.logging.resources.logging"
     }
   ]
 }

--- a/metadata/org.flywaydb/flyway-database-postgresql/index.json
+++ b/metadata/org.flywaydb/flyway-database-postgresql/index.json
@@ -1,17 +1,29 @@
 [
   {
-    "latest": true,
-    "allowed-packages": [
-      "org.flywaydb"
+    "latest" : true,
+    "metadata-version" : "10.19.0",
+    "test-version" : "10.10.0",
+    "source-code-url" : "https://repo1.maven.org/maven2/org/flywaydb/flyway-database-postgresql/$version$/flyway-database-postgresql-$version$-sources.jar",
+    "repository-url" : "https://github.com/flyway/flyway",
+    "test-code-url" : "N/A",
+    "documentation-url" : "https://github.com/flyway/flyway/blob/flyway-$version$/documentation/Flyway%20CLI%20and%20API/Supported%20Databases/PostgreSQL%20Database.md",
+    "description" : "This Flyway module adds PostgreSQL and CockroachDB database support by providing database type detection, SQL parsing, and JDBC integration for PostgreSQL-compatible systems. It enables Flyway migrations to run with PostgreSQL-specific connection handling, schema support, table operations, and advisory locking behavior.",
+    "tested-versions" : [
+      "10.19.0"
     ],
-    "metadata-version": "10.13.0",
-    "source-code-url": "https://repo1.maven.org/maven2/org/flywaydb/flyway-database-postgresql/$version$/flyway-database-postgresql-$version$-sources.jar",
-    "repository-url": "https://github.com/flyway/flyway",
-    "test-code-url": "https://github.com/flyway/flyway/tree/flyway-$version$/flyway-database/flyway-database-postgresql",
-    "documentation-url": "https://github.com/flyway/flyway/tree/flyway-$version$/documentation",
-    "description": "This Flyway module adds PostgreSQL and CockroachDB database support by providing database type detection, SQL parsing, and JDBC integration for PostgreSQL-compatible systems. It enables Flyway migrations to run with PostgreSQL-specific connection handling, schema support, table operations, and advisory locking behavior.",
-    "test-version": "10.10.0",
-    "tested-versions": [
+    "allowed-packages" : [
+      "org.flywaydb"
+    ]
+  },
+  {
+    "metadata-version" : "10.13.0",
+    "test-version" : "10.10.0",
+    "source-code-url" : "https://repo1.maven.org/maven2/org/flywaydb/flyway-database-postgresql/$version$/flyway-database-postgresql-$version$-sources.jar",
+    "repository-url" : "https://github.com/flyway/flyway",
+    "test-code-url" : "https://github.com/flyway/flyway/tree/flyway-$version$/flyway-database/flyway-database-postgresql",
+    "documentation-url" : "https://github.com/flyway/flyway/tree/flyway-$version$/documentation",
+    "description" : "This Flyway module adds PostgreSQL and CockroachDB database support by providing database type detection, SQL parsing, and JDBC integration for PostgreSQL-compatible systems. It enables Flyway migrations to run with PostgreSQL-specific connection handling, schema support, table operations, and advisory locking behavior.",
+    "tested-versions" : [
       "10.13.0",
       "10.14.0",
       "10.15.0",
@@ -24,22 +36,25 @@
       "10.18.0",
       "10.18.1",
       "10.18.2"
+    ],
+    "allowed-packages" : [
+      "org.flywaydb"
     ]
   },
   {
-    "allowed-packages": [
-      "org.flywaydb"
-    ],
-    "metadata-version": "10.10.0",
-    "source-code-url": "https://repo1.maven.org/maven2/org/flywaydb/flyway-database-postgresql/$version$/flyway-database-postgresql-$version$-sources.jar",
-    "repository-url": "https://github.com/flyway/flyway",
-    "test-code-url": "https://github.com/flyway/flyway/tree/flyway-$version$/flyway-database/flyway-database-postgresql",
-    "documentation-url": "https://github.com/flyway/flyway/tree/flyway-$version$/documentation",
-    "tested-versions": [
+    "metadata-version" : "10.10.0",
+    "source-code-url" : "https://repo1.maven.org/maven2/org/flywaydb/flyway-database-postgresql/$version$/flyway-database-postgresql-$version$-sources.jar",
+    "repository-url" : "https://github.com/flyway/flyway",
+    "test-code-url" : "https://github.com/flyway/flyway/tree/flyway-$version$/flyway-database/flyway-database-postgresql",
+    "documentation-url" : "https://github.com/flyway/flyway/tree/flyway-$version$/documentation",
+    "tested-versions" : [
       "10.10.0",
       "10.11.0",
       "10.11.1",
       "10.12.0"
+    ],
+    "allowed-packages" : [
+      "org.flywaydb"
     ]
   }
 ]

